src/test/java/com/ibm/watson/litelinks/test/LitelinksLauncherTests.java

```diff
@@ -194,6 +194,18 @@ public void external_ssl_ca_test() throws Exception {
         do_ssl_ca_test(arr("trustcerts.path", cert2Path, "key.path", keyPath, "key.certpath", certPath),
                 arr("trustcerts.path", certPath, "key.path", key2Path, "key.certpath", cert2Path));
     }
+    
+    @Test
+    public void external_ssl_nonca_test() throws Exception {
+        final Class<?> C = LitelinksLauncherTests.class;
+        final String keyPath = C.getResource("/nonca.key.pem").getFile();
+        final String key2Path = C.getResource("/nonca.key.pem").getFile();
+        final String certPath = C.getResource("/certs/nonca.cert.pem").getFile();
+        final String cert2Path = C.getResource("/certs/nonca.cert.pem").getFile();
```

Comment:
No need for two pairs of these variables. Just have one and use the same vars in both places below (then it's clearer that they're the same).